doc/installing.rst

```diff
@@ -20,38 +20,58 @@ as well as the following Python packages:
 * `NumPy <http://numpy.org/>`_  >=1.13.3
 * `SciPy <http://scipy.org/>`_  >=1.0.0
 * `NetworkX <http://networkx.github.io/>`_ >=2.0
-* `Tensorflow <https://www.tensorflow.org/>`_ >=1.3,<1.7
+* `Blackbird <https://quantum-blackbird.readthedocs.io>`_ >=0.2.0
 
 
 If you currently do not have Python 3 installed, we recommend `Anaconda for Python 3 <https://www.anaconda.com/download/>`_, a distributed version of Python packaged for scientific computation.
 
-.. note::
-
-    As Tensorflow does not currently support Python 3.7 and above, we are unable to
-    also support Strawberry Fields on Python 3.7.
-
 
 Installation
 ============
 
 Installation of Strawberry Fields, as well as all required Python packages mentioned above, can be installed via ``pip``:
-::
 
-    $ python -m pip install strawberryfields
+.. code-block:: console
 
+    pip install strawberryfields
 
-If you are using the ``tensorflow-gpu`` module for TensorFlow GPU support, you can install the following package for GPU support in Strawberry Fields:
-::
 
-    $ python -m pip install strawberryfields-gpu
+TensorFlow support
+==================
 
+To use Strawberry Fields with TensorFlow, version 1.3 of
```

Comment:
A lot of this is duplicated in the TFBackend file itself. Can we not get sphinx to read that in here?

Reply:
Not that I am aware. If the file was a `rst` file we could, using `.. include::`, but it wouldn't work with autodoc.